src/BlockTypes/ProductRating.php

```diff
@@ -113,14 +113,22 @@ protected function render( $attributes, $content, $block ) {
 			$styles_and_classes            = StyleAttributesUtils::get_classes_and_styles_by_attributes( $attributes, array( 'font_size', 'margin', 'text_color' ) );
 			$text_align_styles_and_classes = StyleAttributesUtils::get_text_align_class_and_style( $attributes );
 
+			$average_rating = $product->get_average_rating();
+			$rating_count   = $product->get_rating_count();
+			$reviews_link   = $product->get_permalink() . '#reviews';
+			$link_label     = __( 'Add review', 'woo-gutenberg-products-block' );
+			$link_html      = '<a class="wc-block-components-product-rating__link" href="' . $reviews_link . '">' . esc_attr( $link_label ) . '</a>';
```

Comment:
It might make sense to escape the `href` here:
```suggestion
			$link_html      = '<a class="wc-block-components-product-rating__link" href="' . esc_url( $reviews_link ) . '">' . esc_attr( $link_label ) . '</a>';;
```